content/20.treatments.md

```diff
@@ -1,2 +1,49 @@
 ## Therapeutics {.page_break_before} 
-This section will include a review of information available about therapeutics.
+Given the rapid predicted spread of the disease, the development of therapeutics will be critical to mitigating its effect on health and the mortality rate.
+Typically, therapeutics can take a few forms.
+First, the treatment and reduction of symptoms can result in the reduction of the severity and risk associated with an active infection.
+Second, the development of antiviral drugs can drive a reduced recovery time for patients by inhibiting the development of the virus once an individual is infected.
+Finally, vaccines present a strategy for bolstering the immune response of the populus broadly to the virus, resulting in a lower rate of infection.
+All three of these strategies have been valuable elements of responses to other viruses, including coronaviruses, and are being investigated by researchers at present.
+
+### Treatment of Symptoms
+COVID-19 is characterized by...
+The most severe and concerning symptoms are typically...
+The symptoms most often regarded as the proximal cause of death from COVID-19 are...
+Other diseases with similar symptoms include XYZ but these diseases may be different because...
+
+So far, some strategies for reducing the severity of symptoms have included...
+
+#### Relevant Mechanisms and Possible Pharmaceutical Interventions
```

Comment:
Maybe make clear that these are mechanisms for symptom management to guide contributors? I'm imagining that after this you have a section for each potential therapeutic that you want to cover in this class of interventions.